### PR TITLE
Update OWIN article to RC1

### PR DESCRIPTION
**Pull request by leonardlabat**
Addresses #932

### CODE REVIEW

**Comment by danroth27** [comment]
Thanks @leonardlabat!

@Tratcher Could you please review these updates for RC1?

@Rick-Anderson @Erikre @rachelappel Please review as well.

File: aspnet/fundamentals/owin/sample/global.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿{
     "projects": [ "src", "test" ],
     "sdk": {
-        "version": "1.0.0-beta5-12103"
+        "version": "1.0.0-rc-final"
```
Comment (Tratcher) [inline aspnet/fundamentals/owin/sample/global.json]: shouldn't that be rc1-final?

Comment (Tratcher) [inline aspnet/fundamentals/owin/sample/global.json]: Actually, make it rc1-update1

**Comment by Tratcher** [comment]
FYI: There's more churn here in RC2: https://github.com/aspnet/Entropy/commit/b0c25149f012f5bff3e6b42d0f74711e095edb47

But the RC1 changes look good. :shipit:

File: aspnet/fundamentals/owin.rst [hunk 2]
```diff
@@ -63,15 +63,15 @@ Using ASP.NET Hosting on an OWIN-based server
...
+`IServerFactory <https://github.com/aspnet/Hosting/blob/1.0.0-rc1/src/Microsoft.AspNet.Hosting.Server.Abstractions/IServerFactory.cs>`_ is an ASP.NET interface that requires an Initialize and a Start method. Initialize must return an instance of `IFeatureCollection <https://github.com/aspnet/HttpAbstractions/blob/1.0.0-rc1/src/Microsoft.AspNet.Http.Features/IFeatureCollection.cs>`_, which we populate with a ``INowinServerInformation`` that simply wraps includes the server's name (but the specific implementation may provide additional functionality). In this example, the ``NowinServerInformation`` class is defined as a private class within the factory, and is returned by ``Initialize`` as required.
```
Comment (Rick-Anderson) [inline aspnet/fundamentals/owin.rst]: `` to method names
is an [ DELETE ASP.NET] interface that requires an ``Initialize`` and a ``Start`` method. ``Initialize`` must return an instance of `IFeatureCollection <https://github.com/aspnet/HttpAbstractions/blob/1.0.0-rc1/src/Microsoft.AspNet.Http.Features/IFeatureCollection.cs>`_, which we populate with a ``INowinServerInformation`` that [DELETE simply[DELETE wraps] includes the server's name ([Delete but] the specific implementation may provide additional functionality).

**Comment by danroth27** [comment]
Thanks! 03e7fd6d913a681d85bc53c68b6635e85c549076

**Pull request closed without merging**